server.js

```diff
@@ -18,6 +18,56 @@ server.get('/notes', (req,res)=> {
     .catch(error=> res.status(500).json(error));
 });
 
```

Comment:
Nice work setting up CRUD functionality.  If you get time, it might be worth breaking out the routes into their own file.  but this works.  Good job.